src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -289,6 +287,8 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
     expressPaymentType,
     shippingRate,
   }: StripeExpressCheckoutElementConfirmEvent) => {
+    window.removeEventListener("beforeunload", preventHardReload)
```

Comment:
should we replace this listener in the catch or otherwise if the confirmation fails?